freebridge-ai/insert_dummy_data.py

```
@@ -0,0 +1,111 @@
+            INSERT INTO projects (employer_id, freelancer_id, job_posting_id, project_name, status, headcount, created_at, updated_at)
+            VALUES (%s, %s, %s, %s, %s, 1, %s, %s)
+            """
+            # 김철수 - 금융권 프로젝트 (완료 -> RAG 검색 대상)
+            cursor.execute(sql_project, (employer_id, freelancer_chulsoo_id, job_bank_id, '금융권 차세대', 'COMPLETED', now, now))
+            
+            # 이영희 - AI 프로젝트 (완료 -> RAG 검색 대상)
+            cursor.execute(sql_project, (employer_id, freelancer_younghee_id, job_ai_id, 'AI 챗봇 개발', 'COMPLETED', now, now))
+
+            print("✅ 테스트 데이터 입력이 완료되었습니다!")
+
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)